apps/pie-docs/src/components/checkbox-group/checkbox-group.json

```
@@ -0,0 +1,5 @@
+{
+    "title": "Checkbox Group",
+    "navKey": "Checkbox Group",
+    "description": "Checkbox Group is a helper component that groups Checkbox components into a visual and functional group."
```

```suggestion
    "description": "A checkbox group is a helper component that groups Checkbox components into a visual and functional group."
```
it would be good to change it to follow the same style with the other components